KFC.Red.Server/KFC.Red.DBate.WebAPI/Controllers/QuestionController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using KFC.Red.ManagerLayer.QuestionManagement;
+using KFC.Red.DataAccessLayer.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace KFC.Red.DBate.WebAPI.Controllers
+{
+    public class QuestionManagementRequest
+    {
+        [HttpBindRequired]
+        public Question Question { get; set; }
+
+        [HttpBindRequired]
+        public int QuestionID { get; set; }
+
+        [HttpBindRequired]
+        public string QuestionString { get; set; }
+    }
+
+    public class QuestionController : ApiController
+    {
+        // http GET/POST/PUT/DELETE methods based on add/update/delete questions
+        // see user controller in SSO
+        [HttpPost]
+        [Route("api/question/add")]
+        public IHttpActionResult AddQuestion([FromBody] QuestionManagementRequest Request)
+        {
+            QuestionManager questionM = new QuestionManager();
+            //Question question = new Question();
+            if (questionM.ExistingQuestion(Request.QuestionString) == true)
```

Comment:
You need more error handling, perhaps create more if statements for error handling if needed and OK() if done succesfully such as adding question.